lms/templates/courseware_search/search_list.underscore

```diff
@@ -9,7 +9,9 @@
 
     <% if (hasMoreResults) { %>
         <a class="search-load-next" href="javascript:void(0);">
-            <%= interpolate(gettext("Load next %s results"), [pageSize]) %>
+            <%= interpolate(
+                gettext("Load next %s results"), [pageSize]
```

Comment:
Er wrong ping - @muhammad-ammar this is NOT a change you made.

cc @dsego : this is a change that YOU made, and something you need to be aware of. Also tagging @martynjames for knowledge